bpf/bpf_lxc.c

```diff
@@ -265,7 +266,7 @@ static inline int ipv6_l3_from_lxc(struct __sk_buff *skb,
 		if (ret != TC_ACT_OK)
 			return ret;
 
-		cilium_dbg_capture(skb, DBG_CAPTURE_DELIVERY, HOST_IFINDEX);
+		send_trace_notify(skb, TRACE_FROM_LXC, SECLABEL, 0, 0, HOST_IFINDEX);
```

Comment:
`TRACE_TO_PROXY`

Reply:
Could it be better to include both source and destination in the observation point names, e.g. `TRACE_LXC_TO_PROXY`?

Reply:
Done.